pom.xml

```diff
@@ -16,7 +16,7 @@
         <zentity.website>https://zentity.io</zentity.website>
         <zentity.version>1.6.1</zentity.version>
         <!-- dependency versions -->
-        <elasticsearch.version>7.10.2</elasticsearch.version>
```

Comment:
Is this downgrade a typo?

Reply:
No, I had to downgrade to <7.9.0 because the tests were failing with the multi-node cluster (as expected currently). Which also means the checks for this PR probably won't pass. I'll need to merge this with the PR on async actions before we see it fully pass.

Reply:
Got it, well that's great that the cluster config is catching the bug then! Woo!